docker/docker-compose.yml

```diff
@@ -19,8 +19,11 @@ services:
       # Do not edit the next line. If you want to change the media storage location on your system, edit the value of UPLOAD_LOCATION in the .env file
       - ${UPLOAD_LOCATION}:/usr/src/app/upload
       - /etc/localtime:/etc/localtime:ro
-    env_file:
-      - .env
+    environment:
```

Comment:
If we remove the env_file here, how can the `${UPLOAD_LOCATION}` get its value?

Reply:
Hey Alex, the UPLOAD_LOCATION is only used in the compose file, and the .env file is always available in the compose file. The env_file directive passes those variables *into* the container, UPLOAD_LOCATION is not needed inside the container.

Reply:
I see, so this is a breaking changes for people that adding additional environment variables in our extended list, correct?

Reply:
Some optional env vars would require setting in the `environment:` block. such as 
`IMMICH_TRUSTED_PROXIES` if used.